src/yaml-lint.php

```diff
@@ -82,52 +82,56 @@
     if (count($argPaths) < 1) {
         throw new UsageException('no input specified', EXIT_ERROR);
     }
-    if (count($argPaths) > 1) {
-        throw new UsageException('multiple input files currently unsupported', EXIT_ERROR);
-    }
-
-    $argPath = $argPaths[0];
-
-    if ($argPath === '-') {
+	
+	$lintPath = function($path) use ($argQuiet, $appStr) {
```

Comment:
just moved the already existing code into this closure to ease re-use (and dont define global symbols, like functions or similar)